frameworks/Swift/vapor/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# install swiftenv for managing Swift versions
+eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
```

Comment:
Oh that's weird, I just checked the Travis build and it seems to be installing fine there. Not sure what's up with my local version then.